requirements.txt

```diff
@@ -3,3 +3,4 @@ django-leaflet>=0.23,<0.27
 channels>=2.1.3,<2.5.0
 Pillow>=6.2.0,<=7.1.0
 openwisp-utils>=0.4.0,<0.5.0
+geopy>=1.16.0,<1.21.1
```

Comment:
why are you using 1.16 minimum? If there's no specific reason use 1.20.0 as a minimum please.